pycaret/internal/tabular.py

```diff
@@ -2922,6 +2924,10 @@ def create_model_supervised(
         If not None, will use this dataframe as training target.
         Intended to be only changed by internal functions.
 
+    return_train_score: bool, default = False
+        If not False, will evaluate the train value scores.
+        Intended to be fed as an input from the user.
```

Comment:
Same changes as above in all places where this is documented.

Reply:
Sure @ngupta23. Thanks for the comments.